packages/experimental/TabList/src/Tab/TabTokens.ts

```diff
@@ -0,0 +1,141 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { TabTokens } from '..';
+
+export const tabStates: (keyof TabTokens)[] = [
+  'small',
+  'medium',
+  'large',
+  'vertical',
+  'hovered',
+  'disabled',
+  'selected',
+  'focused',
+  'pressed',
+  'transparent',
+  'subtle',
+];
+
+export const defaultTabTokens: TokenSettings<TabTokens, Theme> = (t: Theme) =>
+  ({
+    indicatorOrientation: 'horizontal',
+    indicatorThickness: 2,
+    borderWidth: 2,
+    borderRadius: 4,
+    contentMarginHorizontal: 2,
+    flexDirection: 'column',
+    borderColor: t.colors.transparentStroke,
+    color: t.colors.neutralForeground2,
```

Comment:
Should a default backgroundColor also be set? Like `backgroundColor: t.colors.neutralBackground1,`?

Reply:
The background color is always gonna be determined by the appearance, which defaults to transparentBackground. No point in setting one here if it's just gonna get overridden.

Reply:
I see, thanks!